### DISCUSSION TITLE
Can't make the text box narrow enough

### DISCUSSION THREAD

**Discussion by Kelvets** [SubtitleEdit/subtitleedit#7654] [Q&A]
![Screenshot_225](https://github.com/SubtitleEdit/subtitleedit/assets/3800465/c5e5e08d-ba5e-4672-b8a3-7d7f7c058ab7)

In the above screenshot I want the translated text to be in a narrower box, so that it's able to fit over the original text. However, even when setting the left and right margins to the maximum allowed (1000), it does not become sufficiently narrow. I believe this started happening after I updated from version 3.6.x to 4.0.* Anyone know a way to produce a narrower text box through a different method?

 /* = I remember that previously I had set a margin size of 700 and it worked. I tried downloading the portable editions of 3.6.x versions but they don't work correctly (the text is not applying the correct style and position).

**Comment by niksedk** [comment]
It's really hard to say without the subtitle file... also, would you allow a wider box? Or a smaller font size?

**Reply by Kelvets** [reply]
I've sent the files to your e-mail under the title "Can't make text box narrow enough - file". A wider box doesn't seem feasible, as it would overlay the video. A smaller font size could work. Thanks.
